Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -191,7 +194,7 @@ - (void)setAttributedText:(NSAttributedString *)attributedText
           [self.backedTextInputView positionFromPosition:self.backedTextInputView.beginningOfDocument offset:newOffset];
       [self.backedTextInputView setSelectedTextRange:[self.backedTextInputView textRangeFromPosition:position
                                                                                           toPosition:position]
-                                      notifyDelegate:YES];
+                                      notifyDelegate:!self.backedTextInputView.ghostTextChanging];
```

Comment:
Can we mark this diff? I know it's in the iOS block but it's still a diff in our repo